bridges/snowbridge/primitives/router/src/inbound/v2.rs

```diff
@@ -163,11 +177,7 @@ where
 		let appendix = vec![
 			RefundSurplus,
 			// Refund excess fees to the relayer
-			// TODO maybe refund all fees to the relayer instead of just DOT?
-			DepositAsset {
-				assets: Wild(AllOf { id: AssetId(fee_asset.into()), fun: WildFungible }),
-				beneficiary: origin_account_location,
-			},
+			DepositAsset { assets: Wild(All), beneficiary: origin_account_location },
```

Comment:
I was wondering if there is any danger of the refund being the end user's assets? Or is it impossible because the refund is always just leftover fee assets, not transferred assets?

Reply:
Your concern here make sense. 

> any danger of the refund being the end user's assets

It depends on how user build the inner xcm. `assets: Wild(AllCounted(8))` in your case may not be correct.

So I think it's better to explicitly only refund fees to relayer.